libr/bin/format/pyc/marshal.c

```diff
@@ -1046,11 +1047,11 @@ static pyc_object *get_object(RBuffer *buffer) {
 		break;
 	case TYPE_CODE_v0:
 		ret = get_code_object (buffer);
-		ret->type = TYPE_CODE_v0;
+		if (ret) { ret->type = TYPE_CODE_v0; }
 		break;
 	case TYPE_CODE_v1:
 		ret = get_code_object (buffer);
-		ret->type = TYPE_CODE_v1;
+		if (ret) { ret->type = TYPE_CODE_v1; }
```

Comment:
multiline pls

Reply:
you asked me previously to have inline if.